opimodel/fonts.py

```diff
@@ -12,11 +12,15 @@
 
 class Font(object):
 
-    def __init__(self, fontface, size, style=REGULAR, pixels=True):
+    def __init__(self, name=None, fontface='Liberation Sans',
```

Comment:
Perhaps 'Arial' is a better default font as it's very likely that every system will have that or a compatible fallback installed. Liberation Sans is Diamond's style guide approved default.